### PR TITLE
include rollback task in running tasks, sort rollback options

### PR DESCRIPTION
**Pull request by anotherchrisberry**
@ajordens please review

### CODE REVIEW

File: app/scripts/modules/core/cluster/cluster.service.js [hunk 1]
```diff
@@ -132,11 +132,23 @@ module.exports = angular.module('spinnaker.core.cluster.service', [
       'enableservergroup': baseTaskMatcher,
       'enablegoogleservergroup': baseTaskMatcher,
       'disablegoogleservergroup': baseTaskMatcher,
+      'rollbackServerGroup': function(task, serverGroup) {
+        var account = task.getValueFor('credentials'),
+            region = task.getValueFor('regions') ? task.getValueFor('regions')[0] : null;
+
+        if (account && serverGroup.account === account && region && serverGroup.region === region) {
+          return serverGroup.name === task.getValueFor('serverGroupName') ||
+            serverGroup.name === task.getValueFor('targetop.asg.enableServerGroup.name');
```
Comment (ajordens) [inline app/scripts/modules/core/cluster/cluster.service.js]: Do we need a match for the `disableServerGroup` side?

enableServerGroup -> disableServerGroup?

File: app/scripts/modules/core/cluster/cluster.service.js [hunk 2]
```diff
@@ -132,11 +132,23 @@ module.exports = angular.module('spinnaker.core.cluster.service', [
...
+        return false;
+      },
     };
 
     function taskMatches(task, serverGroup) {
       let notificationType = _.has(task, 'execution.stages') ?
-        task.execution.stages[0].context['notification.type'] :
+        task.execution.stages[0].context['notification.type'] ?
+          task.execution.stages[0].context['notification.type'] :
+          task.execution.stages[0].type : // TODO: good grief
```
Comment (ajordens) [inline app/scripts/modules/core/cluster/cluster.service.js]: Not sure if this is really particularly terrible... most of the time (all?) notification.type is essentially the stage type.

File: app/scripts/modules/core/cluster/cluster.service.js (hunk 1 above)
Comment (anotherchrisberry) [inline app/scripts/modules/core/cluster/cluster.service.js]: I will update - that reads cleaner than the 'serverGroupName' (just saw that one first and had the value I was expecting)

**Comment by ajordens** [comment]
LGTM

**Pull request merged**